### PR TITLE
Fix mobile display behaviour on dashboard

### PR DESCRIPTION
**Pull request by phirework**
* Cleaned up a bunch of mobile behaviours, including a larger range of sizes for tablet view 
* Got rid of the use of `isNarrow` function on `WeeklyChallengeBoard` - as far as I can tell, it was basically replicating the function of media queries, and event listeners on every resize (esp without throttling) is pretty expensive. I'd like to remove these entirely at some point, but figured that wasn't P0 for challenge stuff
* Created a simplified version of the percentage circle to replace point 2
* Got rid a bunch of absolute heights on the containers for tablet/mobile down. 

I've been testing these mostly by visually comparing with what's on prod. There's some pieces that are slightly adjusted compared to prod, but I decided to prioritize simplicity of code over pixel perfection.

### CODE REVIEW

File: web/src/components/pages/dashboard/challenge/weekly-challenge.css [hunk 1]
```diff
@@ -73,15 +75,15 @@
...
+        padding: 12px 20px;
         font-family: var(--base-font-family);
         font-size: 12px;
         & p:last-child {
             display: none;
         }
         & img {
             float: left;
-            margin-right: 5px;
+            margin: 0.5rem 0.5rem 0 0;
```
Comment (rileyjshaw) [inline web/src/components/pages/dashboard/challenge/weekly-challenge.css]: Nice! Thanks for removing random `px` values where able.

File: web/src/components/pages/dashboard/challenge/weekly-challenge-board.tsx [hunk 2]
```diff
@@ -44,86 +47,81 @@ export default function WeeklyChallengeBoard({
...
-                    <span>{team.invite / team.invite_total}</span>%
-                  </div>
-                  <div className="divider"></div>
-                </React.Fragment>
-              )}
-              <p className="team-invite-total">
-                of {team.invite_total} invites
-              </p>
+                <CircleProgressSimple
+                  numerator={(100 * team.invite) / team.invite_total}
```
Comment (rileyjshaw) [inline web/src/components/pages/dashboard/challenge/weekly-challenge-board.tsx]: Nitpick: we should update these props to match the signature of `CircleProgress`.

**Comment by rileyjshaw** [comment]
Updated to use our existing `Fraction` component:

Screen Shot 2019-11-12 at 3 48 11 PM

**Comment by phirework** [comment]
You're the best, thank you!

**Pull request merged**